TabLayout/TabLayout.js

```diff
@@ -265,6 +273,33 @@ const TabLayoutBase = kind({
 						forward('onExpand', ev, props);
 					}
 				}
+			} else if (is('enter')(keyCode) && !collapsed && document.querySelector(`[data-spotlight-id='${spotlightId}-tabs-expanded']`).contains(target) && target.tagName !== 'INPUT') {
+				Spotlight.setPointerMode(false);
+				ev.preventDefault();
+				let moveTo;
+				if (orientation === 'vertical') {
+					if (anchorTo === 'left') {
+						moveTo = 'right';
+					} else if (anchorTo === 'right') {
+						moveTo = 'left';
+					} else if (anchorTo === 'start') {
+						if (rtl) {
+							moveTo = 'left';
+						} else {
+							moveTo = 'right';
+						}
+					} else if (anchorTo === 'end') {
+						if (!rtl) {
+							moveTo = 'left';
+						} else {
+							moveTo = 'right';
+						}
+					}
```

Comment:
I think it can be modified with conditional chains
```suggestion
	moveTo = anchorTo === 'left' ? 'right'
			: anchorTo === 'right' ? 'left'
			: anchorTo ==='start' ^ rtl ? 'right' : 'left';
```